### PR TITLE
Added descriptor.

### PR DESCRIPTION
**Pull request by simonmika**

### CODE REVIEW

File: Payment/Base.ts [hunk 1]
```diff
@@ -6,19 +6,21 @@ import { Status } from "./Status"
...
 	status: Status
-	reference?: string
 }
 
 export namespace Base {
 	export function is(value: any | Base): value is Base {
 		return typeof(value) == "object" &&
 			Type.is(value.type) &&
+			(typeof(value.reference) == "string" || value.reference == undefined) &&
 			(value.client == undefined || authly.Identifier.is(value.client)) &&
```
Comment (fredrikbryntesson) [inline Payment/Base.ts]: typeof is not a function

File: Payment/CreatableBase.ts [hunk 2]
```diff
@@ -16,7 +17,8 @@ export namespace CreatableBase {
 			Type.is(value.type) &&
 			(value.client == undefined || authly.Identifier.is(value.client)) &&
 			(value.number == undefined || typeof(value.number) == "string") &&
-			isoly.Currency.is(value.currency)
+			isoly.Currency.is(value.currency) &&
+			(value.descriptor == undefined || typeof(value.descriptor) == "string")
 	}
```
Comment (fredrikbryntesson) [inline Payment/CreatableBase.ts]: typeof is not a function

File: Payment/CreatableBase.ts [hunk 3]
```diff
@@ -27,6 +29,7 @@ export namespace CreatableBase {
 					value.client == undefined || authly.Identifier.is(value.client) || { property: "client", type: "authly.Identifier | undefined" },
 					value.number == undefined || typeof(value.number) == "string" || { property: "number", type: "string | undefined" },
 					isoly.Currency.is(value.currency) || { property: "currency", type: "isoly.Currency" },
+					value.descriptor == undefined || typeof(value.descriptor) == "string" || { property: "descriptor", type: "string | undefined" },
 				].filter(gracely.Flaw.is) as gracely.Flaw[],
```
Comment (fredrikbryntesson) [inline Payment/CreatableBase.ts]: typeof is not a function

File: Payment/Base.ts [hunk 4]
```diff
@@ -27,7 +29,7 @@ export namespace Base {
 			isoly.Currency.is(value.currency) &&
 			(typeof(value.verify) == "string" || value.verify == undefined) &&
 			(typeof(value.token) == "string" || value.token == undefined) &&
-			Status.is(value.status) &&
-			(typeof(value.reference) == 'string' || value.reference == undefined)
-	}
+			(typeof(value.descriptor) == "string" || value.descriptor == undefined) &&
+			Status.is(value.status)
```
Comment (fredrikbryntesson) [inline Payment/Base.ts]: typeof is not a function

**Comment by simonmika** [comment]
@fredrikbryntesson Happy?

**Pull request merged**